go/vt/sqlparser/sql.y

```diff
@@ -3740,6 +3740,14 @@ index_option:
   {
     $$ = &IndexOption{Name: string($1), Value: NewStrVal($2)}
   }
+| ENGINE_ATTRIBUTE equal_opt ID
```

Comment:
test for index option? also does this one only support ID?

Reply:
Oops this is supposed to be only STRING, not ID.